### PR TITLE
feat: upgrade app packages

### PR DESCRIPTION
**Pull request by nlewis84**
This is ready to go now. The below note has been fixed.

```
** Note for myself - merge the divergent branches into this one first before you merge this to master **

@redreceipt No review needed, but I do have a question for you from the server caching you added like 10 days ago. 

Check the lines that I commented out in `src > client > index.js`. If I don't comment them out, I cannot get past login and here is the error that I see. Am I missing something locally to get this working? Commenting out these lines allows the app to work as intended with no obvious side effects.
![Screen Shot 2022-01-13 at 4 43 42 PM](https://user-images.githubusercontent.com/72768221/149420883-7d1a3eb0-a306-4190-bc1a-690b1fe8f607.png)

```

### CODE REVIEW

File: src/client/index.js [hunk 1]
```diff
@@ -1,14 +1,18 @@
 import { useEffect } from 'react';
 import PropTypes from 'prop-types';
 import { ApolloProvider, ApolloClient, ApolloLink, gql } from '@apollo/client';
+// import { createPersistedQueryLink } from '@apollo/client/link/persisted-queries';
+// import { sha256 } from 'react-native-sha256';
```
Comment (nlewis84) [inline src/client/index.js:5]: These lines

File: src/client/index.js [hunk 2]
```diff
@@ -37,9 +41,33 @@ const onAuthError = async () => {
...
+// Android's emulator requires localhost network traffic to go through 10.0.2.2
+if (Platform.OS === 'android') {
+  uri = uri.replace('localhost', androidUri);
+}
+
 const errorLink = buildErrorLink(onAuthError);
+// const apqLink = createPersistedQueryLink({
+//   sha256,
+//   useGETForHashedQueries: true,
+// });
```
Comment (nlewis84) [inline src/client/index.js:56]: These and then where we pass in apqLink below

File: src/client/index.js [hunk 3]
```diff
@@ -37,9 +41,33 @@ const onAuthError = async () => {
...
 
-const link = ApolloLink.from([authLink, errorLink, httpLink]);
+const link = ApolloLink.from([
+  authLink,
+  errorLink,
+  // apqLink,
```
Comment (nlewis84) [inline src/client/index.js:61]: Last commented line

File: src/client/index.js (hunk 1 above)
Comment (redreceipt) [reply src/client/index.js:5]: is `react-native-sha256` added to the pkg json?

Comment (nlewis84) [reply src/client/index.js:5]: Yeah, 1.4.7

**Comment by redreceipt** [comment]
Then you got some debug to do ;) we can pair when you’re fed up

**Comment by nlewis84** [comment]
https://user-images.githubusercontent.com/72768221/150866217-cfd7dbbc-ea04-43a8-bf35-22cac6e075ef.mp4

**Pull request merged**